Cargo.toml

```diff
@@ -37,15 +37,15 @@ async-signature = "0.3.0"
 base64 = "0.13"
 pem-rfc7468 = "0.7.0"
 x509-cert = { version = "0.1.1", features = ["pem"] }
+coset = { version = "0.3.7", features = ["std"] }
+ciborium = "0.2.2"
```

Comment:
tried to minimize the changes, as would be a lot of them to change all calls to `ciborium`. But there are interoperability issues so we might end-up doing that.

Reply:
I'll try to migrate all to `ciborium` and not use `serde` at all.

Reply:
@cobward @timothee-haudebourg I'm almost done migrating to ciborium, there was a lot of changes, but it's finish soon.

I used the coset traits for serialize ciborium, as they use it in the `CoseSign1`, works good with tagged. But found they also have a serializare compatible with serde https://docs.rs/ciborium/latest/ciborium/ser/fn.into_writer.html but the problem is it's doesn't write tagged, but strangely it reads from tagged. I didn't found any way to make it work

I think it would be nice to have some macros to serialize serde objects with ciborium but to also support tagged. I think would benefit the serde ecosystem to have a crate like that.
Maybe after I'm done with the migration we could work in this?

Reply:
migrated to ciborium